### DISCUSSION TITLE
Newbie to MDX. Need to create/find MDX code for a simple Feedback form.

### DISCUSSION THREAD

**Discussion by russurquhart1** [mdx-js/mdx#2237] [Q&A] [answered]
Hi All,
I'm putting together our website in Docusaurus. I need to add a simple user feedback form. I was told that I could use MDX to create such a form. Has anyone done this and/or have some MDX code that does this, or has some of the functionality and I could probably add to it? I just need to be able to have the users enter their name, email address, and comments. When they submit the feedback have it email to our docs email address.

Any help is greatly appreciated. I know that MDX is powerful, but I'm still coming up to speed on it!

Russ

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @russurquhart1! 👋 
MDX allows you access to components in your framework of choice. In your case Docusaurus/React.
There isn't an MDX specific solution, per se.
The solution would be use your favorite React form library/solution, and import it into your MDX documents.

You could create your own form with plain React https://reactjs.org/docs/forms.html and your own backend to store responses.
You could use any React form library https://blog.gandi.net/en/posts/form-lib-comparison/ and your own backend.
Or you could use one of the many form services which offer React support for the frontend and backend https://css-tricks.com/a-comparison-of-static-form-providers/
